src/menu/src/MenuOption.js

```diff
@@ -53,6 +52,7 @@ const MenuOption = memo(function MenuOption(props) {
       height={40}
```

Comment:
Can we push `height`, `display='flex'`, and `alignItems='center'` into the theme config?